features/0335-http-over-didcomm/README.md

```diff
@@ -77,16 +81,16 @@ DIDComm messages for this protocol look as follows:
 {
   "@type": "https://didcomm.org/http-over-didcomm/1.0/request",
   "@id": "2a0ec6db-471d-42ed-84ee-f9544db9da4b",
-  "~purpose": [<partial resource uri: scheme://userinfo@host:port/path>],
+  "~purpose": [],
   "method": <method>,
   "resource-uri": <resource uri value>,
```

Comment:
`resource-uri` can also be optional in case where agent wants to hide identity of http server.

Reply:
Done.